frontend/src/metabase/visualizations/lib/table.js

```diff
@@ -91,7 +91,9 @@ export function getTableHeaderClickedObject(
   } else {
     return {
       column,
-      dimension: query?.dimensionForColumn(column),
+      dimension:
```

Comment:
should it be a ternary here so that dimension is not boolean when it is not function?